### ISSUE TITLE
HIP 63: Helium Hub

### ISSUE THREAD

**Issue by hiptron** [helium/HIP#423]
- Author(s): @anthonyra 
- Start Date: 2022-05-26 
- Category: Economic
- Original HIP PR: https://github.com/helium/HIP/pull/419
- Tracking Issue: https://github.com/helium/HIP/issues/423

# Rendered View

Read the HIP:

https://github.com/helium/HIP/blob/main/0063-helium-hub.md

# Summary
[summary]: #summary

This HIP is an alternative to the one being presented as [HIP-51 Helium DAO](0051-helium-dao.md). This HIP will take the goverance proposal with slight modifications from that HIP but drastically changes the economic and technical implementation. As Helium Hub the Helium blockchain will become the center/hub of a web of wireless protocols all of which have the ability to leverage the Helium Ecosystem.

**Comment by TheRealJohnMac50** [comment]
A number of pointers, my apologies for the ones that have already been answered. 

- I'm all for both the entire PoPS and bonding pool concepts, especially with the maintaining $21,000,000 in DC. It's economically sound, I see that you may have taken economics 101, or even comprehend it naturally. For PoPS, I'm a fan of the conceps to apply DC to DC rewards, but what sort of proof-of coverage will we have that someone actually passed the DC? Also Important to clarity, do you intend for the bonding pool's $21,000,000 DC requirement and PoPS to apply retroactively, as in for the MOBILE and other subDAOs that may arise prior to the completion of this HIP?

 - How does the age of a subDAO factor into the rewards per epoch that the subDAO may receive? 

 - Your veHNT voting system is absolutely brilliant! It ensures that voting power must be provided to those willing to lock up their HNT the longest. However, a potential downside, is that it may enable those with the largest HNT wallets absolute domination. Arguably, in essence, the result would most likely be that we hand the vast majority of voting power over to the top 10 largest HNT wallets. They have many millions worth of HNT, and if they only lock up even just 10% of their HNT for the longest possible duration, they would still have hundreds of millions worth of veHNT. 

 - Your drawbacks section that you detailed are obviously some pretty big drawbacks :) Can't imagine what it would feel like to archive all of the Erlang work accomplished over the years. Additionally, with all of the current ongoing unprecedented developments that are of immediate urgency, it would certainly be some time from now before it would be possible to achieve this core code language and economical restructure. Which gives up plenty of time to perfect this :)
 
In conclusion, there are certainly various concepts that I'm absolutely all far, and this is clearly a very thought out and brilliant HIP.  Well done and thank you for your contributions to the community!

**Comment by vincenzospaghetti** [comment]
Hi @Anthonyra, it looks like this HIP is stale and is replaced by the approved HIP 51 and HIP 70. We are closing this effective immediately.
